isis/src/docsys/build/UserExplore.xsl

```diff
@@ -145,7 +145,7 @@ Deborah Lee Soltesz
           <td style="width:50%">
             <dl>
               <dt><h4>About ISIS Applications</h4></dt>
-              <dd><a href="https://usgs-astrogeology.github.io/ISIS3/ISIS_Cube_Format.html">
+              <dd><a href="https://isis.astrogeology.usgs.gov/documents/CommandLine/CommandLine.html">
```

Comment:
This looks to be the oddball link pointing to the original site. Should this page be tagged to be moved to GitHub?

Reply:
I'm not seeing a direct analogue to this page on the github pages.  The closest thing I can find is https://github.com/DOI-USGS/ISIS3/wiki/Introduction_to_ISIS#Reserved-Command-Line-Arguments-   This section and the subsequent sections seem to do a pretty good job of covering command line usage.

Reply:
Even if it's not a direct copy of the previous docs, I think https://github.com/DOI-USGS/ISIS3/wiki/Introduction_to_ISIS#Reserved-Command-Line-Arguments- is where we should be linking.  I'd say for consistency's sake we should be targeting all GitHub docs.

If the information at the github location is not sufficient, then content/documentation should be added to the new docs rather than directing users to the old docs.

Reply:
I might be misunderstanding, should this link be changed again or is it good?